packages/components/src/components/notifications/Toast/useToast.test.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { screen } from '@tests/renderWithProviders';
+import { renderHook, act } from '@testing-library/react-hooks';
+import { ComposerProvider } from '@theme';
+import { useToast } from './useToast';
+
+const wrapper = ({ children }) => <ComposerProvider>{children}</ComposerProvider>;
+
+describe('useToast', () => {
+  test('should render custom component when provided', () => {
+    const { result } = renderHook(() => useToast(), { wrapper });
+    act(() => {
+      result.current({ title: 'foo', render: () => <div>foo</div> });
+    });
+    screen.getByText(/foo/i);
+  });
+
+  test('should render Alert by default when render is not provided', () => {
+    const { result } = renderHook(() => useToast(), { wrapper });
+    act(() => {
+      result.current({ title: 'title' });
+    });
+    screen.getByTestId('cmpsr.alert');
```

Comment:
I see why you added the id in the `Alert`, I will add this data-testid in the `useToast` instead